src/Locations.ts

```diff
@@ -11,6 +9,7 @@ import type { SpaceMember } from './types.js';
 import type { PresenceMember } from './utilities/types.js';
 import type Space from './Space.js';
 import { ERR_NOT_ENTERED_SPACE } from './Errors.js';
+import SpaceUpdate from './SpaceUpdate.js';
```

Comment:
I'm confused, why do imports have extensions? Is that normal?

Reply:
Have a look at this issue https://github.com/microsoft/TypeScript/issues/16577#issuecomment-703190339 and our own here https://github.com/ably/spaces/issues/76. `.js` extensions provide the widest compatibility when loading in different environments, after bundling.